src/python_pachyderm/mixin/auth.py

```diff
@@ -177,7 +186,8 @@ def authenticate_oidc(self, oidc_state: str) -> str:
         str
             A token that can be used for making authenticate requests.
         """
-        return self._req(Service.AUTH, "Authenticate", oidc_state=oidc_state).pach_token
+        message = auth_pb2.AuthenticateRequest(oidc_state=oidc_state)
+        return self.__stub.Authorize(message).pach_token
```

Comment:
This should be `self.__stub.Authenticate()`. Might be good to get coverage on these methods in the tests.

Reply:
Good catch, I'd definitely like to increase the coverage here, but I'm not exactly sure of a great way to test it. any ideas?